src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -103,27 +179,95 @@ nrays(PF::PolyhedralFan) = pm_fan(PF).N_RAYS
 ## Points properties
 ###############################################################################
 
-"""
+@doc Markdown.doc"""
     lineality_space(PF::PolyhedralFan)
 
-Returns the lineality_space of a polyhedral fan.
+Return a non-redundant matrix whose rows are generators of the lineality space
+of `PF`.
+
+# Examples
+This fan consists of two cones, one containing all the points with $y ≤ 0$ and
+one containing all the points with $y ≥ 0$. The fan's lineality is the common
+lineality of these two cones, i.e. in $x$-direction.
+```jldoctest
+julia> PF = PolyhedralFan([1 0; 0 1; -1 0; 0 -1], IncidenceMatrix([[1, 2, 3], [3, 4, 1]]))
+A polyhedral fan in ambient dimension 2
+
+julia> lineality_space(PF)
+pm::Matrix<pm::Rational>
+1 0
+
+```
 """
 lineality_space(PF::PolyhedralFan) = pm_fan(PF).LINEALITY_SPACE
 
 
 """
     rays_as_point_matrix(PF::PolyhedralFan)
 
-Returns the rays of a polyhedral fan as rows of a matrix.
+Return the rays of `PF` as rows of a matrix.
+
+# Examples
+The rays of a face fan (centered at the origin) point into the direction of the
+polytope's vertices; here we see this for the `3`-cube.
```

Comment:
```suggestion
polytope's vertices; here we see this for the 3-cube.
```
At least that would match how it is written elsewhere. You can of course also change it everywhere else to use `3`-cube, but it seems a bit odd (I could understand `$3$-cube`, though, but I don't think there'll be a visual difference in the resulting HTML?)